README.md

```diff
@@ -1,2 +1,3 @@
-# ae-design-system
-Our design system implementation. This repo outputs a Storybook site as well as an NPM package of reusable components and tokens.
+# Design System
+
+This repo contains our primary design system implementation. For now, it contains a basic React template [generated using Vite](https://vitejs.dev/guide/#scaffolding-your-first-vite-project) with the `react-ts` template.
```

Comment:
```suggestion
This repo contains the primary design system for DCP applications. For now, it contains a basic React template [generated using Vite](https://vitejs.dev/guide/#scaffolding-your-first-vite-project) with the `react-ts` template.
```

Reply:
I can appreciate that my wording here is a little opaque but I don't think we should say that this repository _is_ the design system. The standards, components, guidelines, and best practices that make up a design system exist outside of this repo.